src/js/components/Sidebar.js

```diff
@@ -163,7 +163,11 @@ class Sidebar extends React.Component {
       }
 
       if (group.category !== "root") {
-        heading = <h3 className="sidebar-section-header">{group.category}</h3>;
+        heading = (
+          <Trans render="h3" className="sidebar-section-header">
```

Comment:
For this to do the translation I think we need to give `{group.category}` to `Trans` as `id` prop